src/panels/config/integrations/integration-panels/zwave_js/dialog-zwave_js-remove-node.ts

```diff
@@ -58,15 +73,23 @@ class DialogZWaveJSRemoveNode extends LitElement {
   public async showDialog(
     params: ZWaveJSRemoveNodeDialogParams
   ): Promise<void> {
-    this.entry_id = params.entry_id;
-    this._removedCallback = params.removedCallback;
-    if (params.skipConfirmation) {
+    this._entryId = params.entryId;
+    this._deviceId = params.deviceId;
+    this._onClose = params.onClose;
+    if (this._deviceId) {
+      const nodeStatus = await fetchZwaveNodeStatus(this.hass, this._deviceId!);
+      this._device = this.hass.devices[this._deviceId];
+      this._step =
+        nodeStatus.status === NodeStatus.Dead ? "start_removal" : "start";
```

Comment:
Meaning that if the status is not dead, it can still be dead that we need to force remove it?

Reply:
Yes. It can take hours to for a dead device to show as such